### ISSUE TITLE
Safari in Q&A

### ISSUE THREAD

**Issue by AustinConlon** [missing-semester/missing-semester#123]
I think that the [last part of the Q&A](https://github.com/missing-semester/missing-semester/blob/master/_2020/qa.md#any-comments-on-differences-between-web-browsers) undersells the merits of Safari. Firefox excels for privacy reasons as you mention, but so does Safari. The lower memory footprint and battery life usage are benefits of Safari that aren't mentioned.

**Comment by jonhoo** [comment]
My personal take on this is that these days there is very little reason to use Safari over Chrome or Firefox. @JJGO @anishathalye ?

**Comment by JJGO** [comment]
It's not so clear-cut for me. On macOS Chrome eats up ridiculous amounts of RAM whereas Safari is noticeably more efficient. 

Regarding privacy, both support uBlock Origin which does most of the heavy lifting and both lack neater features like Multi-Account Containers. Chrome also has a track record of not respecting privacy when it comes to Google sites [source](https://news.ycombinator.com/item?id=24817304).

I also like Safari's compatibility. Whenever a website is giving me trouble because of my browser setup in Firefox, I'll use Safari and it works 100% of time.

TL;DR: If I had to stop using Firefox, I'll probably use Safari as my default

**Comment by anishathalye** [comment]
We currently say:

> Should you want an alternative, Firefox is our recommendation.

Should we switch to Firefox as the recommended browser? I personally use Firefox, Jose does too, and iirc Jon does as well?

It might also be worth mentioning Chromium, the slightly de-Google'd version of Chrome (though it's missing things like automatic updates). I currently use Chromium on the rare occasion where a site that I need to use does not work properly in Firefox.

Also, like @AustinConlon suggested, maybe we could say a bit more about Safari being a privacy-respecting browser. (I don't know as much about the memory/battery story of Safari vs Firefox currently, but if there's data on this, we can mention/link it too).

**Comment by mahmud2011** [comment]
> I also like Safari's compatibility. Whenever a website is giving me trouble because of my browser setup in Firefox, I'll use Safari and it works 100% of time.

For me it's the other way around.
